### DISCUSSION TITLE
CentOS7 Version?

### DISCUSSION THREAD

**Discussion by mistermcdoogle** [swampapp/swamp#41] [General]
Is there a yum package for this? I'd like to try swamp but I use CentOS7
This appears to be Debian only if I am correct

**Comment by rubiojr** [comment]
@jmacdoug007 I added RPMs somewhat recently, but you'll need to build it yourself for now. I haven't tested if building actually works in CentOS 7, which could be a bit dated for all this to work. I've been using Fedora 34/35 to test mainly.
